### PR TITLE
Feat/recommender components

### PR DESCRIPTION
**Pull request by scopalaffairs**
# People Discovery and Recommendation Components

Services and Views for

* Trending Users
* Recommended Users
* Similar Users

### CODE REVIEW

File: src/LatestPeople.jsx [hunk 1]
```diff
@@ -70,26 +90,62 @@ const ButtonLink = styled.a`
...
+        />
+      </Select>
+    </FlexContainer>
+    {state.selectedView.value === "latest" && (
+      <>
+        <LatestPeople>
+          {accounts.map((account) => (
+            <Item key={account.accountId}>
+              <Widget
+                src="near/widget/AccountProfile"
```
Comment (shelegdmitriy) [inline src/LatestPeople.jsx:120]:
```suggestion
                src="${REPL_ACCOUNT}/widget/AccountProfile"
```

File: src/PeoplePage.jsx [hunk 2]
```diff
@@ -275,36 +277,74 @@ return (
...
-      </Items>
+    {(state.selectedTab == "everyone" ||
+      state.selectedTab == "following" ||
+      state.selectedTab == "followers") &&
+      items.length > 0 && (
+        <Items>
+          {items.map((person, i) => (
+            <Item key={person.accountId}>
+              <Widget
+                src="near/widget/AccountProfileCard"
```
Comment (shelegdmitriy) [inline src/PeoplePage.jsx:309]:
```suggestion
                src="${REPL_ACCOUNT}/widget/AccountProfileCard"
```

File: src/Recommender/Account/AccountProfileLargeCard.jsx [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+      <TextLink href={profileUrl} ellipsis bold>
+        {props.profileName || profile.name || accountId.split(".near")[0]}
+      </TextLink>
+      <TextLink href={profileUrl} ellipsis>
+        @{accountId}
+      </TextLink>
+    </CenteredLinksWrapper>
+    {tags.length > 0 ? (
+      <TagsWrapper>
+        <Widget src="near/widget/Tags" props={{ tags, scroll: true }} />
```
Comment (shelegdmitriy) [inline src/Recommender/Account/AccountProfileLargeCard.jsx:185]:
```suggestion
        <Widget src="${REPL_ACCOUNT}/widget/Tags" props={{ tags, scroll: true }} />
```

File: src/Recommender/Account/AccountProfileViewSwitch.jsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+          followers: props.followers || null,
+          following: props.following || null,
+          profileImage: props.profileImage || null,
+          profileName: props.profileName || null,
+          scope: props.scope || null,
+        }}
+      />
+    )}
+  </>
+);
```
Comment (shelegdmitriy) [inline src/Recommender/Account/AccountProfileViewSwitch.jsx:43]:
```suggestion

const forwardedProps = {
  accountId: props.accountId,
  sidebar: props.sidebar || null,
  followsYou: props.followsYou || null,
  becauseYouFollow: props.becauseYouFollow || null,
  likers: props.likers || null,
  followers: props.followers || null,
  following: props.following || null,
  profileImage: props.profileImage || null,
  profileName: props.profileName || null,
  scope: props.scope || null,
};

return (
  <>
    {props.sidebar ? (
      <Widget
        src="${REPL_ACCOUNT}/widget/AccountProfile"
        props={{
          ...forwardedProps,
          overlayPlacement: "left",
        }}
      />
    ) : (
      <Widget
        src="${REPL_ACCOUNT}/widget/Recommender.Account.AccountProfileLargeCard"
        props={{ ...forwardedProps }}
      />
    )}
  </>
);
```

**Review by shelegdmitriy — CHANGES_REQUESTED** [review]

**Comment by scopalaffairs** [comment]
https://github.com/near/near-discovery-components/pull/373
has all changes, so we can close this one.

**Pull request closed without merging**